pages/image-converter.tsx

```diff
@@ -0,0 +1,195 @@
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import Container from '@mui/material/Container';
+import FormControl from '@mui/material/FormControl';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import FormLabel from '@mui/material/FormLabel';
+import Grid from '@mui/material/Grid';
+import Radio from '@mui/material/Radio';
+import RadioGroup from '@mui/material/RadioGroup';
+import Slider from '@mui/material/Slider';
+import { styled } from '@mui/material/styles';
+import Typography from '@mui/material/Typography';
+import React, { useState } from 'react';
+
+import Heading from '../components/Heading';
+import Layout from '../components/Layout';
+
+enum FileType {
+  PNG = 'image/png',
+  JPG = 'image/jpeg',
+  WEBP = 'image/webp',
+}
+
+const FILE_TYPE_EXTENSIONS = {
+  [FileType.PNG]: '.png',
+  [FileType.JPG]: '.jpg',
+  [FileType.WEBP]: '.webp',
+};
+
+const FILE_TYPE_OPTIONS = [
+  { label: 'PNG', value: FileType.PNG },
+  { label: 'JPG', value: FileType.JPG },
+  { label: 'WebP', value: FileType.WEBP },
+];
+
+function renameFile(filename: string, fileType: FileType) {
+  return (
+    filename.substring(0, filename.lastIndexOf('.')) +
+    FILE_TYPE_EXTENSIONS[fileType]
+  );
+}
+
+function formatSliderLabel(value: number) {
+  return `${Math.round(value * 100)}%`;
+}
+
+const PreviewImage = styled('img')({});
+
+export default function JsonPage() {
+  const [file, setFile] = useState<{ raw: File; imageSrc: string }>();
+  const [fileType, setFileType] = useState<FileType>(FileType.WEBP);
+  const [quality, setQuality] = useState(0.7);
+  const handleInputChange: React.ChangeEventHandler<
+    HTMLInputElement
+  > = (event) => {
+    // Bail immediately if the user canceled
+    if (
+      !event.currentTarget.files ||
+      event.currentTarget.files.length === 0
+    ) {
+      return;
+    }
+    const eventFile = event.currentTarget.files[0];
+    const reader = new FileReader();
+    console.log(eventFile);
```

Comment:
Please remove the console log here

Reply:
My bad! Do you think it'd be helpful to enable [`no-console`](https://eslint.org/docs/latest/rules/no-console)?

Reply:
Yes, it was enabled before but I made some changes and must've clobbered that lol